Makefile

```diff
@@ -89,6 +90,16 @@ else
   REPRO :=
 endif
 
+# Release build requires using parent dir as base for buildroot
+RELEASE_BUILD := $(test -f "$(BASE_DIR)/.gitignore" ; echo $?)
```

Comment:
I tried this locally but it didn't work. Spent some time debugging it, found out that this solves the problem:

```
RELEASE_BUILD := $(shell test -f "$(BASE_DIR)/.gitignore" ; echo $$?)
```

Reply:
Good catch, fixed.